src/components/editable-input/editable-input.jsx

```diff
@@ -66,6 +67,16 @@ class EditableInput extends React.Component {
     this.setState({ editing: false });
   };
 
+  renderHelpText() {
+    const modifiers = {
+      hasSuccess: this.props.hasSuccess,
+      hasWarning: this.props.hasWarning,
+      hasError: this.props.hasError,
+    };
+
+    return !this.props.helpText ? null : <HelpText {...modifiers}>{this.props.helpText}</HelpText>;
```

Comment:
This logic might be better to put inside the `<HelpText />` component (to avoid this extra function)